### DISCUSSION TITLE
Alternate Energy Dashboard (With SoC + Power)

### DISCUSSION THREAD

**Discussion by BigManDave** [macxq/foxess-ha#23] [Show and tell]
If anyone is interested I removed the default HA Energy dashboard and created a new UI Dashboard for Energy. Might save others a bit of googling :)
NOTE: entity 'sensor.foxess_pv_power_total' will need to be changed depending on how/if you've totalled the separate PV Power entities.
![image](https://user-images.githubusercontent.com/66840216/149328762-0f3dede4-b3d6-4af6-a18d-ea8b392b7a1c.png)

views:
  - title: Home
    type: sidebar
    layout: {}
    icon: ''
    badges: []
    cards:
      - type: grid
        cards:
          - type: energy-usage-graph
          - type: energy-solar-graph
          - type: energy-sources-table
        columns: 1
        square: false
      - type: grid
        cards:
          - type: energy-date-selection
          - type: energy-distribution
          - type: entities
            entities:
              - entity: sensor.foxess_grid_consumption_power
              - entity: sensor.foxess_pv_power_total
              - entity: sensor.foxess_bat_charge_power
              - entity: sensor.foxess_bat_discharge_power
              - entity: sensor.foxess_feedin_power
              - entity: sensor.foxess_load_power
              - entity: sensor.foxess_bat_temperature
          - type: gauge
            entity: sensor.foxess_bat_soc
            min: 10
            max: 100
            needle: true
            severity:
              green: 70
              yellow: 25
              red: 0
            name: Battery
          - type: energy-grid-neutrality-gauge
          - type: energy-solar-consumed-gauge
          - type: energy-carbon-consumed-gauge
        columns: 1
        square: false
        view_layout:
          position: sidebar
      - type: energy-devices-graph
title: Energy

**Comment by Thebig24fan** [comment]
Great thanks I’ll try it when get fox bit of ha setup as It’s only new second day still s bit confused
